rosdep/python.yaml

```diff
@@ -8768,6 +8768,17 @@ python3-pytest-mock:
       packages: [pytest-mock]
   rhel: ['python%{python3_pkgversion}-pytest-mock']
   ubuntu: [python3-pytest-mock]
+python3-pytest-order:
+  arch: [python-pytest-order]
+  debian: [python3-pytest-order]
+  gentoo: [dev-python/pytest-order]
+  nixos: [python3Packages.pytest-order]
+  opensuse: [python3-pytest-order]
+  ubuntu:
+    '*': [python3-pytest-order]
+    focal:
+      pip:
+        packages: [pytest-order]
```

Comment:
Is this how we should handle focal as the oldest active LTS?

Reply:
Yes.  If you don't care about focal working at all, you can also do:

```
focal: null
```

Reply:
Simplified.